### DISCUSSION TITLE
Delete old environment

### DISCUSSION THREAD

**Discussion by zzulueta** [microsoft/AI-Chat-App-Hack#53] [Q&A] [answered]
I did a list of my environments and got this:
zzulueta ➜ /workspaces/azure-search-openai-demo (main) $ azd env list
NAME          DEFAULT   LOCAL     REMOTE
ragchatapp    false     true      false
ragchatapp01  true      true      false
ragchatapp02  false     true      false

I want to delete the old ones but there does not seem to be any command on this?

Usage
  azd env [command]

Available Commands
  get-values    : Get all environment values.
  list          : List environments.
  new           : Create a new environment and set it as the default.
  refresh       : Refresh environment settings by using information from a previous infrastructure provision.
  select        : Set the default environment.
  set           : Manage your environment settings.

Flags
        --docs  : Opens the documentation for azd env in your web browser.
    -h, --help  : Gets help for env.

Global Flags
    -C, --cwd string    : Sets the current working directory.
        --debug         : Enables debugging and diagnostics logging.
        --no-prompt     : Accepts the default value instead of prompting, or it fails if there is no default

**Comment by cjrieth** [comment]
In order to delete an environment I have just been deleting the corresponding env folder within in the `/.azure` folder. So for environment `ragchatapp` I would delete `/.azure/ragchatapp`.

There may be some better way to do a clean environment delete, and if there is I'd love to know!

**Reply by pamelafox** [reply, marked as answer]
Hm, I would first do "azd down" to delete the resource group, and _then_ delete the folder. But yeah maybe azd should add an "azd env delete". 
You could file a feature request in https://github.com/azure/azure-dev
